temperature/utils.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+import httpx
+from fastapi import HTTPException
+
+from settings import settings
+
+
+WEATHER_URL = (
+    f"http://api.weatherapi.com/v1/current.json?key={settings.WEATHER_TOKEN}"
```

Comment:
Ensure that the 'WEATHER_TOKEN' environment variable is correctly set. This is crucial for accessing the weather API and fetching temperature data.